README.md

```diff
@@ -51,7 +53,7 @@ If this gives you unexpected Cordova errors, run:
 
     npm run clean && npm ci && npm run action gulp -- build android
 
-Cordova will generate a new Android project in the platforms/android directory.  Install the built apk by  platforms/android/build/outputs/apk/android-armv7-debug.apk
+Cordova will generate a new Android project in the platforms/android directory. Install the built apk by `platforms/android/app/build/outputs/apk/<processor>/debug/app-<processor>-debug.apk`
```

Comment:
it would be nice to have a way to easily detect the correct processor, or just mention where to find it

Reply:
added a hint, but will let the developer to figure it out.